pyproject.toml

```diff
@@ -63,9 +63,12 @@ simplejson = ">=3.17.0"
 # Optional dependencies for markup extra
 anymarkup-core = { version = ">=0.8.0", optional = true }
 xmltodict = { version = ">=0.12.0", optional = true }
+# Optional dependencies for shell functions integration tests
+pytest-container = {version = "*", optional = true}
```

Comment:
I meant that this should still be in the `tool.poetry.group.test.dependencies` section.

Reply:
done